framework/src/transfers/MultiAppTransfer.C

```diff
@@ -609,3 +609,24 @@ MultiAppTransfer::checkVariable(const FEProblemBase & fe_problem,
       paramError(param_name, "The variable '", var_name, "' does not exist.");
   }
 }
+
+Point
+MultiAppTransfer::getPointInTargetAppFrame(const Point & p,
+                                           unsigned int local_i_to,
+                                           const std::string & phase) const
+{
+  const auto & to_transform = _to_transforms[getGlobalTargetAppIndex(local_i_to)];
+  if (to_transform->hasCoordinateSystemTypeChange())
+  {
+    if (!_skip_coordinate_collapsing)
+      mooseInfo(phase + " cannot use the point in the target app frame to the "
```

Comment:
"app frame to the" -> "app frame due to the"

Reply:
changed